src/main/java/com/Elessar/app/util/Metric.java

```diff
@@ -0,0 +1,35 @@
+package com.Elessar.app.util;
+
+/**
+ * Created by Hans on 2/16/19.
+ */
+public class Metric {
```

Comment:
Some recommendations:

1. Instead of passing a MetricManager to the constructor you could let the MetricManager create the Metric object:
```
final Metric metric = MetricManager.newMetric("server.db.update");
```

2. I suggest we get rid of timerStart() function all together and instead start the timer inside Metric's constructor. The benefits are:
- we are guaranteed that the startTime is valid up on Metric object creation
- as a consequence, timerStop() no longer need to throw exception, which means we can get rid of all those try-catch around timerStart() and timerStop() calls

Reply:
I'm thinking in reality all MetricManager methods and variables should be defined as static since it is shared by all classes from one program, while Metric is instance based and will be instantiated once per method. Is it right?

Reply:
Right now we kind of hack the logger inside MetricManager so that client and server write to different metric log files. But yes, the MetricManager class probably only need static functions.